packages/client/src/generation/TSClient/utils/invalidImportWarning.ts

```diff
@@ -9,10 +9,11 @@ export const invalidImportWarningTs = [
   ' - For pnpm, run `pnpm add db@link:./path/to/your/client`',
   ' - For yarn, run `yarn add db@link:./path/to/your/client`',
   '-----',
-  '3. If you have a `tsconfig.json`, set `moduleResolution` to `node16`, `nodenext`, or `bundler`.',
-  'Alternatively, if you prefer, setting `module` to `node16` or `nodenext` equally works.',
```

Comment:
We don't need to mention `Node16`, it's deprecated and `NodeNext` is equivalent to it (as per TS [docs](https://www.typescriptlang.org/tsconfig#node16nodenext)), less is better here.

Reply:
Also, now we are pointing users to set both rather than explaining all possible and invalid combinations, and when  and when it is not ok to have just the one or the other.